oresat_dxwifi/data/oresat_dxwifi.eds

```diff
@@ -4191,6 +4192,13 @@ ParameterName=Temperature
 ObjectType=0x07
 DataType=0x0008
 AccessType=ro
+DefaultValue=0
+
+[6002]
+ParameterName=New Variable
+ObjectType=0x07
+DataType=0x0007
+AccessType=rw
```

Comment:
`DefaultValue` and `PDOMapping` are missing here. compared to the dcf file. Do they belong here?

Reply:
File no longer necessary.